source/templates/search.liquid

```diff
@@ -1,8 +1,8 @@
 <h1>Search</h1>
 
-<form action="/search" method="get" class="search form" role="search">
+<form action="/search" method="get" class="search form">
 
-  <div class="form__group">
+  <div class="form__group" role="search">
```

Comment:
I found this article: http://adrianroselli.com/2015/08/where-to-put-your-search-role.html